examples/PathHandler.js

```diff
@@ -377,7 +379,7 @@ PathHandler.prototype.curvetoQuadraticSmoothRel = function(x, y) {
 
     this.addShape(Shapes.quadraticBezier(
         controlX, controlY,
-        this.lastX + x, this,lastY + y
+        this.lastX + x, this, this.lastY + y
```

Comment:
Looks like I typed a comma instead of a period. So `this.` needs to be deleted and `,` needs to be `.`

Reply:
Ah, my apologies, fixed.